openmetadata-service/src/main/java/org/openmetadata/service/search/indexes/ContainerIndex.java

```diff
@@ -78,6 +78,9 @@ public Map<String, Object> buildESDoc() {
             ? 0
             : container.getVotes().getUpVotes() - container.getVotes().getDownVotes());
     doc.put("domain", getEntityWithDisplayName(container.getDomain()));
+    doc.put(
+        "descriptionStatus",
+        getDescriptionStatus(container.getDescription(), container.getDataModel().getColumns()));
```

Comment:
Should we move this to the parent class to avoid duplicating the code for all EntityInterface object and just call that method from here? On the SearchIndex side we could do something like

```
try {
     Method method = entity.getClass().getMethod("getColumns");
     List<Column> cols = ((List<Column>) method.invoke(entity);
     ...
catch (NoSuchMethodException, ... e) {
    ...
}
```

Reply:
cc @mohityadav766 any thought if this makes sense ^

Reply:
@IceS2 @TeddyCr yup it does make sense , we can move this along with other attribute that are available across Assets implementing EntityInterface to have default behaviour in base class, only entity specific fields can be updated in here

Reply:
@IceS2 we can have buildESDoc as a default methods, and at the end call addEntitySpecificDataToDoc , where we can add other attribute that are specfic to entityType

Reply:
Just pushed some changes that go into that direction!